### ISSUE TITLE
ROS2 colcon build fails

### ISSUE THREAD

**Issue by BartVanderHaagen** [rust-lang/cargo#11730]
### Problem

The package requires the Cargo feature called `workspace-inheritance`, but that feature is not stabilized in this version of Cargo (1.61.0)

what version does not have this problem? and why is there nowhere a explanation to be found? 

### Version

```text
colcon build --packages-select zenoh_bridge_dds --cmake-args -DCMAKE_BUILD_TYPE=Release
Starting >>> zenoh_bridge_dds
--- stderr: zenoh_bridge_dds                              
error: failed to parse manifest at `/home/ubuntu/zenoh-plugin-dds/zenoh-bridge-dds/Cargo.toml`

Caused by:
  feature `workspace-inheritance` is required

  The package requires the Cargo feature called `workspace-inheritance`, but that feature is not stabilized in this version of Cargo (1.61.0).
  Consider adding `cargo-features = ["workspace-inheritance"]` to the top of Cargo.toml (above the [package] table) to tell Cargo you are opting in to use this unstable feature.
  See https://doc.rust-lang.org/nightly/cargo/reference/unstable.html#workspace-inheritance for more information about the status of this feature.
gmake[2]: *** [CMakeFiles/build_crate.dir/build.make:81: target/release/zenoh-bridge-dds] Error 101
gmake[1]: *** [CMakeFiles/Makefile2:137: CMakeFiles/build_crate.dir/all] Error 2
gmake: *** [Makefile:146: all] Error 2
---
Failed   <<< zenoh_bridge_dds [0.22s, exited with code 2]

Summary: 0 packages finished [0.53s]
  1 package failed: zenoh_bridge_dds
  1 package had stderr output: zenoh_bridge_dds
ubuntu@ip-172-31-22-118:~/zenoh-plugin-dds$
```

**Comment by Eh2406** [comment]
The explanation can be found two lines further down. It recommends you read https://doc.rust-lang.org/nightly/cargo/reference/unstable.html#workspace-inheritance and that link does include that the feature was stabilized in 1.68. When the version of cargo you are using was released it did not include more information because it was not yet available. When 1.61 came out we did not know when the feature would be stabilized.

**Comment by weihanglo** [comment]
There is a link inside the error message: <https://doc.rust-lang.org/nightly/cargo/reference/unstable.html#workspace-inheritance>. There are more resources out there. The feature has been stabilized since Rust 1.64. 

If the error message is not clear enough for people to find what they want, I am really sorry I cannot ride a way back machine to amend that message in the already released Rust 1.61 😢.

**Comment by weihanglo** [comment]
This is soon to be closed, as Eh2406 has already pointed out the situation and answered the questions. Thank you!
